geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ListIndexCommand.java

```diff
@@ -80,6 +84,10 @@ public ResultModel listIndex(@CliOption(key = CliStrings.LIST_INDEX__STATS,
         indexTable.accumulate("Keys", adapter.getNumberOfKeys());
         indexTable.accumulate("Values", adapter.getNumberOfValues());
       }
+
+      if (maintStatus) {
+        indexTable.accumulate("Index Maintenance Status", indexDetails.getIndexMaintenance());
```

Comment:
Why not simply always display this info?

Reply:
I was concerned about backward compatibility.

Reply:
What happens if an old gfsh is used on a newer system?  There will definitely be serialization issues, whether it matters or not is a different story I guess

Reply:
Gfsh is restricted to connecting to an instance with the same `major.minor` version.

@mivanac The gfsh output is not guaranteed to be stable so adding this here is OK (i.e remove this check). Generally, it's also much safer to add information as opposed to removing it.

Reply:
Just confirming what @jdeppe-pivotal said 👍